crates/matrix-sdk-crypto/src/identities/user.rs

```diff
@@ -406,19 +425,22 @@ impl ReadOnlyUserIdentity {
     /// * `master_key` - The master key of the user identity.
     ///
     /// * `self signing key` - The self signing key of user identity.
+    /// * `tofu_trusted` - Is this identity tofu trusted.
```

Comment:
If we're sticking with a single flag, we will need to explain what "tofu trusted" actually means each time we reference it.